gb-cpu/src/processor/cpu/registers/area.rs

```diff
@@ -0,0 +1,46 @@
+#[derive(Debug)]
+pub enum _8Bits {
```

Comment:
No reasons at all except that i can't start with a number and i find it repetitive to have the enum named RegisterXBits in a mod named register already. When you use it you reapeating: like "register::Register16Bits" and that's what im unhappy about

The other enum is not in this place and there is a slight difference in the 16bit register as AF isnt a real 16 bit register, and the SP and PC don't have to be treated in a special way

Reply:
you can rename them
```rust
use foo::foobar as bar;
```

Reply:
If there's no choice that's a completly valid solution, but i rather have directly 

`use foo::bar` 

if we can afford it, which here I think we can. 

Plus i mainly use those enum to implement the get and set traits so we can either have

```
use register::Register16bits as ???
use register::Register8bits as !!!

impl Get<???> for Register
impl Get<!!!> for Register
```

Whatever the ??? and !!! stands for, i'd rather have it directly be the name of the enum, so we can just use

`use register::*`

I don't want to use it as is because it goes:

```
use register::*

impl Get<Register16Bits> for Register
```

And that repetition i'm unhappy about having register repeated like that

Reply:
1. I do not like the use of wildcard, It hide part of code. 
2. But you could rename `Register16Bits` into `Reg16` directly where it's define

Reply:
Starting a variable/function name by `_` is a convention for unused code. Maybe we can rename the existing enums into `Bits8` and `Bits16` ?

Reply:
`Bits{8,16}` are also a good scheme since we import it from `gb_roms::register::Bits8`

Reply:
I agree with the removal of the wildcard, explicit is way better.

For the definition, that's a point we should discuss about code organisation: I'm really for having only one struct or a few enum per file.

And I learned about the convention afterwards, so Bits16 and Bits8 are good suggestions for me